incident-bot.json

```diff
@@ -406,7 +406,7 @@
                 "export PAGERDUTY_SERVICE_API_KEY=", { "Ref" : "ENVPagerDutyServiceKey" },"\n",
                 "export PAGERDUTY_SUBDOMAIN=", { "Ref" : "ENVPagerDutySubDomain" },"\n",
                 "export PAGERDUTY_USER_ID=", { "Ref" : "ENVPagerDutyUserID" },"\n",
-                "export PAGERDUTY_ROOM=", { "Ref" : "ENVPagerDutyRoom" },"\n",
+                "export HUBOT_INCIDENT_PAGERDUTY_ROOM=", { "Ref" : "ENVPagerDutyRoom" },"\n",
```

Comment:
changing it here is only half the steps needed, you also have to modify the userdata.sh